src/fragments/lib/auth/android/signin_next_steps/10_signin.mdx

```diff
@@ -4,9 +4,78 @@ The `nextStep` property is of enum type `AuthSignInStep`. Depending on its value
+                                Log.i(TAG, "SMS code sent to "+nextStep.getCodeDeliveryDetails().getDestination());
+                                Log.i(TAG, "Additional Info :"+ nextStep.getAdditionalInfo());
+                                // Prompt the user to enter the SMS MFA code they received
+                                // Then invoke `confirmSignIn` api with the code
+                                break;
+                            }
+                            case CONFIRM_SIGN_IN_WITH_CUSTOM_CHALLENGE: {
+                                Log.i(TAG, "Custom challenge, additional info: "+ nextStep.getAdditionalInfo());
+                                // Prompt the user to enter custom challenge answer
+                                // Then invoke `confirmSignIn` api with the answer
+                                break;
+                            }
+                            case CONFIRM_SIGN_IN_WITH_NEW_PASSWORD: {
+                                Log.i(TAG, "Sign in with new password, additional info: "+ nextStep.getAdditionalInfo());
+                                // Prompt the user to enter a new password
+                                // Then invoke `confirmSignIn` api with new password
+                                break;
+                            }
+                            case RESET_PASSWORD: {
+                                Log.i(TAG, "Reset password, additional info: "+ nextStep.getAdditionalInfo());
+                                // User needs to reset their password.
+                                // Invoke `resetPassword` api to start the reset password
+                                // flow, and once reset password flow completes, invoke
+                                // `signIn` api to trigger signIn flow again.
+                                break;
+                            }
+                            case CONFIRM_SIGN_UP: {
+                                Log.i(TAG, "Confirm signup, additional info: "+ nextStep.getAdditionalInfo());
+                                // User was not confirmed during the signup process.
+                                // Invoke `confirmSignUp` api to confirm the user if
+                                // they have the confirmation code. If they do not have the
+                                // confirmation code, invoke `resendSignUpCode` to send the
+                                // code again.
+                                // After the user is confirmed, invoke the `signIn` api again.
+                                break;
+                            }
+                            case DONE: {
+                                Log.i(TAG, "SignIn complete");
+                                // User has successfully signed in to the app
+                                break;
+                            }
+                        }
+                    }
+                    ,
```

Comment:
This comma can be moved up to line 64 after the `}`

Reply:
done